src/daemon/instance_settings_handler.cpp

```diff
@@ -118,11 +118,9 @@ mp::MemorySize get_memory_size(const QString& key, const QString& val)
 void update_cpus(const QString& key, const QString& val, mp::VirtualMachine& instance, mp::VMSpecs& spec)
 {
     bool converted_ok = false;
-    if (auto cpus = val.toInt(&converted_ok); !converted_ok || cpus < 1)
+    if (auto cpus = val.toInt(&converted_ok); !converted_ok || cpus < std::stoi(mp::min_cpu_cores))
         throw mp::InvalidSettingException{key, val, "Need a positive integer (in decimal format)"};
```

Comment:
Can we improve the error message? It's not anymore just a positive integer, the message can be something like "Less than %1 CPUs are not allowed."

Reply:
Hey @luis4a0!

I'm not sure what you mean.  Nothing really changed here other than I use the already defined `min_cpu_cores` rather than the hardcoded `1`, but it still only accepts a positive integer (in decimal format).

Reply:
Well, if `min_cpu_cores` is set for whatever reason to 2, the current error message will not be confusing.

Reply:
Hmm, I see where you are coming from, but that would also apply if the original hard coded `1` was changed to a hard coded `2`.  But yeah, perhaps that error message could be changed a bit to reflect whatever `min_cpu_cores` value is.  You convinced me to change it :slightly_smiling_face:

Reply:
Ok, changed the message slightly, but left the other wording intact since it's possible to pass in values such as `1.5`, `foo`, etc., so need to be explicit about it being a positive integer in decimal format.